### PR TITLE
Display info when there are no functions to show in Outline

### PR DESCRIPTION
**Pull request by nyrosmith**
Associated Issue: #3895 

Outline now displays "No functions" when there are no functions to show in outline.
I only did visual testing.

I also moved the query if Outline is enabled into PrimaryPane. There is no need to Render Outline at all if it is disabled.

### CODE REVIEW

File: src/components/PrimaryPanes/Outline.js [hunk 1]
```diff
@@ -51,23 +58,23 @@ export class Outline extends Component {
...
-    }
+    const { isHidden, symbols } = this.props;
+
+    const symbolsToDisplay = symbols.functions
+      .filter(func => func.name != "anonymous");
 
     return (
       <div className={classnames("outline", { hidden: isHidden })}>
-        <ul className="outline-list">{this.renderFunctions()}</ul>
+        { symbolsToDisplay.length >0 ? this.renderFunctions(symbolsToDisplay) : this.renderPlaceholder() }
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/Outline.js]: `symbolsToDisplay.length > 0`

**Comment by nyrosmith** [comment]
outline

**Comment by jasonLaster** [comment]
the CI issues look like the branch needs a rebase and a fix to flow

File: src/components/PrimaryPanes/index.js [hunk 2]
```diff
@@ -111,6 +111,13 @@ class PrimaryPanes extends Component {
     const { selectedPane } = this.state;
     const { sources, selectSource } = this.props;
 
+    const outlineComp = isEnabled("outline") ? (
+      <Outline
+        selectSource={selectSource}
+        isHidden={selectedPane === "sources"}
+      />
+    ) : null;
```
Comment (bomsy) [inline src/components/PrimaryPanes/index.js]: nice!

File: src/components/PrimaryPanes/Outline.css [hunk 3]
```diff
@@ -2,6 +2,14 @@
   overflow-y: auto;
 }
 
+.outline .outline-pane-info {
+  width: 100%;
+  font-style: italic;
+  text-align: center;
+  padding: 0.5em;
+  user-select: none;
```
Comment (bomsy) [inline src/components/PrimaryPanes/Outline.css]: image

Could we add a `font-size: 12px` so its matches the sizes for `No Breakpoints` and 'Not Paused'.
Atm it looks smaller.

**Comment by bomsy** [comment]
@nyrosmith This looks great! Just one comment and we are good to go! Great Work!

**Comment by nyrosmith** [comment]
@bomsy Done!

**Pull request merged**